src/project_creator/templates/project/{{cookiecutter.name}}/pytest.ini

```diff
@@ -6,5 +6,4 @@ testpaths =
 junit_logging = all

```

Comment:
I do not know why we needed to generate the test-report.xml. Was it for Jenkins to be able to show the test results?